File: config/src/main/java/com/typesafe/config/impl/FullIncluder.java
/**
 *   Copyright (C) 2011-2012 Typesafe Inc. <http://typesafe.com>
 */
package com.twitter_typesafe.config.impl;

import com.twitter_typesafe.config.ConfigIncluder;
import com.twitter_typesafe.config.ConfigIncluderClasspath;
import com.twitter_typesafe.config.ConfigIncluderFile;
import com.twitter_typesafe.config.ConfigIncluderURL;

interface FullIncluder extends ConfigIncluder, ConfigIncluderFile, ConfigIncluderURL,
            ConfigIncluderClasspath {

}

File: config/src/main/scala/com/typesafe/config/impl/FullIncluder.scala
/**
 *   Copyright (C) 2011-2012 Typesafe Inc. <http://typesafe.com>
 */
package com.typesafe.config.impl

import com.typesafe.config.ConfigIncluder
import com.typesafe.config.ConfigIncluderClasspath
import com.typesafe.config.ConfigIncluderFile
import com.typesafe.config.ConfigIncluderURL

trait FullIncluder extends ConfigIncluder
    with ConfigIncluderFile
    with ConfigIncluderURL
    with ConfigIncluderClasspath {}
